test/calendar.test.tsx

```diff
@@ -134,4 +134,62 @@ describe('Calendar', () => {
     expect(onEventClickSpy).toHaveBeenCalledTimes(1)
     expect(onEventClickSpy).toHaveBeenCalledWith(event)
   })
+
+  it('should render the custom Prev, Next, and today buttons', () => {
+    const wrapper = shallow(<Calendar />)
+    const fullCalendar = wrapper.find(FullCalendar)
+    const customButtons = fullCalendar.prop('customButtons') as any
+
+    expect(customButtons.customPrev.text).toEqual('Prev')
+    expect(customButtons.customNext.text).toEqual('Next')
+    expect(customButtons.customToday.text).toEqual('today')
+  })
+
+  it('should call the onPrevClick callback when the custom Prev button is clicked', () => {
+    const onPrevClickSpy = jest.fn()
+    const wrapper = mount(<Calendar onPrevClick={onPrevClickSpy} />)
+    const fullCalendar = wrapper.find(FullCalendar)
+    const customButtons = fullCalendar.prop('customButtons') as any
+    const prevButton = customButtons.customPrev
+
+    act(() => {
+      ;(prevButton.click as any)({
+        el: expect.any(HTMLElement),
+      })
```

Comment:
let's right this as: 

```ts
const onClick = previousButton.click as any
onClick({el: expect.any(HTMLElement});
```

that way we don't get the weird dangling semi-colon

Reply:
✅ in 1f3b3d6